release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -61,9 +61,9 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-15-CAPO-tech-preview-no-upgrade"]
 ==== CAPO integration into the cluster CAPI Operator (Tech Preview)
 
-If you enable the `TechPreviewNoUpgrade` feature flag, the Coherent Accelerator Processor Interface (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifestyle. The CAPI Operator automatically creates an additional OpenStack cluster and an additional {product-title} cluster for the current {product-title} cluster.
+If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster API (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifestyle. The CAPI Operator automatically creates an additional OpenStack cluster and an additional {product-title} cluster for the current {product-title} cluster.
```

Comment:
```suggestion
If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster API (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifestyle. The CAPI Operator automatically creates `Cluster` and `OpenStackCluster` resources for the current {product-title} cluster.
```

Reply:
Note that the `Cluster` and `OpenStackCluster` here are specific API resources provided by CAPI and CAPO respectively. We're *not* talking about "clusters" in the generic sense. As such, I think they need to be formatted as `literals`.